integration-tests/src/test/resources/oke/terraform/cluster.tf

```diff
@@ -2,62 +2,103 @@
+}
+
+variable "node_pool_initial_node_labels_key" {
+  default = "key"
+}
+
+variable "node_pool_initial_node_labels_value" {
+  default = "value"
+}
+
+variable "node_pool_kubernetes_version" {
+  default = "v1.17.9"
 }
 
+variable "node_pool_name" {
+  default = "tfTestCluster_workers"
+}
+
+variable "node_pool_node_image_name" {
+  default = "Oracle-Linux-7.6"
+}
+
+variable "node_pool_node_shape" {
+  default = "VM.Standard2.1"
+}
+
+variable "node_pool_quantity_per_subnet" {
+  default = 2
+}
+
+variable "node_pool_ssh_public_key" {
+}
+
+data "oci_identity_availability_domains" "tfsample_availability_domains" {
+  compartment_id = var.compartment_ocid
+}
 
 resource "oci_containerengine_cluster" "tfsample_cluster" {
   #Required
-  compartment_id = "${var.compartment_ocid}"
-  kubernetes_version = "${var.cluster_kubernetes_version}"
-  name = "${var.cluster_name}"
-  vcn_id = "${oci_core_virtual_network.oke-vcn.id}"
+  compartment_id     = var.compartment_ocid
+  kubernetes_version = var.cluster_kubernetes_version
+  name               = var.cluster_name
+  vcn_id             = oci_core_virtual_network.oke-vcn.id
 
   #Optional
   options {
-    service_lb_subnet_ids = ["${oci_core_subnet.oke-subnet-loadbalancer-1.id}", "${oci_core_subnet.oke-subnet-loadbalancer-2.id}"]
+    service_lb_subnet_ids = [oci_core_subnet.oke-subnet-loadbalancer-1.id, oci_core_subnet.oke-subnet-loadbalancer-2.id]
 
     #Optional
     add_ons {
       #Optional
-      is_kubernetes_dashboard_enabled = "${var.cluster_options_add_ons_is_kubernetes_dashboard_enabled}"
-      is_tiller_enabled = "${var.cluster_options_add_ons_is_tiller_enabled}"
+      is_kubernetes_dashboard_enabled = var.cluster_options_add_ons_is_kubernetes_dashboard_enabled
+      is_tiller_enabled               = var.cluster_options_add_ons_is_tiller_enabled
     }
   }
 }
 
 resource "oci_containerengine_node_pool" "tfsample_node_pool" {
-	#Required
-	cluster_id = "${oci_containerengine_cluster.tfsample_cluster.id}"
-	compartment_id = "${var.compartment_ocid}"
-	kubernetes_version = "${var.node_pool_kubernetes_version}"
-	name = "${var.node_pool_name}"
-	node_image_name = "${var.node_pool_node_image_name}"
-	node_shape = "${var.node_pool_node_shape}"
-        #subnet_ids = ["${oci_core_subnet.oke-subnet-worker-1.id}", "${oci_core_subnet.oke-subnet-worker-2.id}","${oci_core_subnet.oke-subnet-worker-3.id}"]
-        subnet_ids = ["${oci_core_subnet.oke-subnet-worker-1.id}", "${oci_core_subnet.oke-subnet-worker-2.id}"]
+  #Required
+  cluster_id         = oci_containerengine_cluster.tfsample_cluster.id
+  compartment_id     = var.compartment_ocid
+  kubernetes_version = var.node_pool_kubernetes_version
+  name               = var.node_pool_name
+  node_image_name    = var.node_pool_node_image_name
+  node_shape         = var.node_pool_node_shape
+
+  #subnet_ids = ["${oci_core_subnet.oke-subnet-worker-1.id}", "${oci_core_subnet.oke-subnet-worker-2.id}","${oci_core_subnet.oke-subnet-worker-3.id}"]
```

Comment:
delete commented out line

Reply:
I left them intentionally , so we can switch between2 and 3 nodes in cluster